### PR TITLE
Sign up form project: Changed project steps to better reflect expected knowledge

### LINKED ISSUE #23643: Sign-up form project: Incorrectly assumes knowledge about :focus
- [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
- [x] The title of this issue follows the `where the change occurs: brief description of request` format, e.g. `Global SCSS file: Buttons should have XYZ`

- [x] I would like to be assigned this issue to work on it

**1. Description of the Enhancement**

On the [Sign up Form](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/intermediate-html-and-css/lessons/sign-up-form) Project, Step 3.4 requires to use a `focus` class to give a different styling to the existing form (specifically a blue border). It also mentions this isn't best practice, and that you'll learn how to in a later lesson.

This seems incorrect to me, since in the lesson on [Advanced Selectors](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/intermediate-html-and-css/lessons/advanced-selectors) the `:focus` pseudo-class is already mentioned, so knowledge about this subject should be known to the student when starting the sign-up form project.

**2. Additional Information**

A possible addition to the fix would be to link back to the relevant lesson in the assignment, but this wouldn't match the way previous projects work, so my preference / recommendation would be to just mention the fact that you should add the blue border styling to the relevant `:focus` subclass

### PR DESCRIPTION
**Pull request by NotAtec**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #23643

**2. This PR:**

- Changed usage of `focus` class to `:focus` pseudo-class since the student should have knowledge about pseudo-classes before they start this project.
- Broke up the step in 2, to make it easier to understand and match more with the shorter steps in other projects.

**3. Additional Information:**

### CODE REVIEW

File: html_css/project-sign-up-form/project-sign-up-form.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This project is intended to give you a chance to flex some of the new items you'
...
 2. The area behind the "ODIN" logo is a div that has a dark, but semi-transparent background color. This enhances the readability of the text against the busy background image.
 3. The color we've chosen for the 'Create Account' button is similar to tones found in the background image. Specifically, it is `#596D48`.
-4. The inputs, by default have a very light border (`#E5E7EB`), but we've included 2 variations. The "Last Name" input should be given a class of `focus` so that it can be styled with a blue border and a subtle box-shadow. The password inputs should be given an `error` class. Using a class for `focus` isn't best practice, you'll learn how to do that properly in a later lesson.
-5. Do not worry about making your project look nice on mobile, but DO resize your browser a little bit to make sure that it's not completely broken at different desktop resolutions.
+4. The inputs, by default have a very light border (`#E5E7EB`), but we've included 2 variations. The password inputs should be given an `error` class.
```
Comment (dm-murphy) [inline html_css/project-sign-up-form/project-sign-up-form.md:25]:
```suggestion
4. The inputs, by default have a very light border (`#E5E7EB`), but we've included 2 variations. For starters, the password inputs should be given an `error` class.
```

**Review by dm-murphy — CHANGES_REQUESTED** [review]
Thanks for fixing these instructions! I like the breakdown of the steps here overall. But because this separates the two variations into two bullets, I think it needs a little rephrasing for this bullet to indicate this is the first step of two. I added in a suggestion if you'd like to use that or you could come up with another way of phrasing it.

**Comment by NotAtec** [comment]
The suggestion looks good to me, so comitted it as you've phrased it

**Review by dm-murphy — APPROVED** [review]
Thanks!

**Pull request merged**